### ISSUE TITLE
Release 1.0 runtimes

### ISSUE THREAD

**Issue by bkchr** [polkadot-fellows/runtimes#20]
The repository should now be prepared enough to create the first runtime release from this repo. Parity released all crates alongside releasing the 1.0 release of the node. As the 1.0 release is now already out for some time, I propose that we do the runtimes 1.0 release based on the next Parity Polkadot node release.

So, we first need to wait for this next node release to happen. After the release happened, someone can propose a pull request that updates to these crates on crates.io.

CC @ggwpez @coderobe

**Comment by bkchr** [comment]
BTW, this time we don't want to copy the commits as before. From this point on this repo will work independently from the Parity repos when it comes to updating the runtimes. The runtimes in this repo are also the canonical ones.

**Comment by bkchr** [comment]
> o just copying the changes instead of the full history?

Yes



> Replacing the dependencies to use crates-io instead of git? okay.

I mean I hope we get the latest release this week on crates.io, otherwise we need to use git dependencies.

**Comment by acatangiu** [comment]
What’s the timeline here? When can we expect a new runtime release? 

Seeing how node [v1.1.0 is out](https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-v1.1.0) should we not release runtimes 1.1.0 directly (instead of v1.0)?

**Comment by bkchr** [comment]
https://github.com/polkadot-fellows/runtimes/pull/28 it is being worked on here.

**Comment by acatangiu** [comment]
#28 was merged. Can we do a release now? We need a runtime release for [BEEFY on Kusama](https://github.com/orgs/paritytech/projects/27/views/11?pane=issue&itemId=14196332).
Anything I can help with getting this over the finish line?

**Comment by bkchr** [comment]
We need to wait for the staking changes runtime upgrade to be approved. After that is approved, we need to merge this fix here to make the repo canonical again. Then we can bring the release over the finish line. I think we can actually do that before the fix is enacted on chain, but we should be certain that it gets enacted on chain. 

Maybe we can then do the next release faster...

**Comment by acatangiu** [comment]
Ok, thanks for the context! 👍

**Comment by bkchr** [comment]
So, I would like to merge the following pr first: https://github.com/polkadot-fellows/runtimes/pull/63

After this pr is merged, we can update this pr https://github.com/polkadot-fellows/runtimes/pull/26 and then finally do the release.

**Comment by joepetrowski** [comment]
Also required:

- [ ] https://github.com/polkadot-fellows/runtimes/pull/46
- [ ] https://github.com/paritytech/polkadot-sdk/pull/1733 (IMO this is a short term solution but will allow backwards compatibility for some XCM programs already being sent)
- [ ] Fix failing integration tests that causes parachains to stall @NachoPal

**Comment by kianenigma** [comment]
- [ ] https://github.com/paritytech/polkadot-sdk/issues/125
- [ ] https://github.com/polkadot-fellows/runtimes/pull/26

**Comment by NachoPal** [comment]
Reason why Parachains stall after relay chain runtime upgrade: https://github.com/paritytech/polkadot-sdk/issues/1923#issuecomment-1768016015

**Comment by kianenigma** [comment]
No action item needed for https://github.com/paritytech/polkadot-sdk/issues/125 in this release. 

Context: `VoteLockingPeriod` was mistakenly set to 7 days, and in this release it is going back to 28, the value that it should be. This will not impact the user's voting power. At the moment, the lock can be removed sooner than expected in Polkadot. Once we fix this, the locks will be according to the spec. Implementing https://github.com/polkadot-fellows/RFCs/pull/20#issuecomment-1673553108 is a different concern and probably will need consideration, but is outside the scope of this release.

**Comment by bkchr** [comment]
We should not forget to bump the `spec_version`s.

**Comment by bkchr** [comment]
No this will not be included anymore. This can go into the next release. We want to start with the next release shortly after this was has finally be dine (which is hopefully today).

**Comment by bkchr** [comment]
To not break every downstream user as we have renamed the `xcm` crate to `staging-xcm` to have it released on crates.io, I have created the following two prs:
- https://github.com/paritytech/polkadot-sdk/pull/1948
- https://github.com/paritytech/scale-info/pull/197

With these two prs we will have `TypeInfo::path` not return `staging-xcm` as one segment and instead `xcm`. So, the metadata type name shouldn't change and downstream users should not need to change anything. A proper solution is being discussed here: https://github.com/polkadot-js/api/issues/5741 TLDR: Do not reference the type by name as there might be dragons ;)

We will require that both prs are merged and the appropriate crates being relased. For the `staging-xcm` crate we will need to backport the changes to `1.1.0` and `1.2.0` and release a patch release of the `staging-xcm` crate.
